### ISSUE TITLE
Rings only once on Jellybean & Samsung Galaxy S3

### ISSUE THREAD

**Issue by GoogleCodeExporter** [abedmaatalla/sipdroid#1074]
```
This is a duplicate of 1059 that incorrectly got merged with 295
295 states that it works if you put a long ringtone on this does NOT work in 
this case. The ring disappears the instant that the screen alert appears.
Any chance of moving this to High please

Beta 2.7 on Android 4.1.1

```

Original issue reported on code.google.com by `sjha...@gmail.com` on 7 Dec 2012 at 9:21
